skfda/ml/regression/_linear_regression.py

```diff
@@ -150,6 +158,55 @@ class LinearRegression(
         >>> linear.predict([x, x_fd])
         array([ 11.,  10.,  12.,   6.,  10.,  13.])
 
+        Funcionality with pandas Dataframe.
+
+        First example:
+
+        >>> x_basis = Monomial(n_basis=3)
+        >>> cov_list = [ ( FDataBasis(x_basis, [0, 0, 1]) ),
+        ...              ( FDataBasis(x_basis, [0, 1, 0]) ),
+        ...              ( FDataBasis(x_basis, [0, 1, 1]) ),
+        ...              ( FDataBasis(x_basis, [1, 0, 1]) )]
+        >>> y = [2, 3, 4, 5]
+        >>> df = pd.DataFrame(cov_list)
```

Comment:
This is wrong. Each covariate should be a different column, and the rows should be different samples.